gemini-agent-harness-engineering-guide.html

```
@@ -0,0 +1,2271 @@
+<span class="kw">class</span> <span class="fn">FakeLlm</span>(BaseLlm):
+    <span class="str">"""
+    テスト用フェイクLLM。
+    事前に定義したシナリオ（query → response）に従って
+    決定論的な応答を返す。APIコスト・ネットワーク接続なし。
+    """</span>
+
+    <span class="kw">def</span> <span class="fn">__init__</span>(self, scenarios: dict[str, LlmResponse]):
+        self._scenarios = scenarios
+        self._call_history: list[str] = []  <span class="cmt"># 呼び出し記録（検証用）</span>
+
+    <span class="kw">async def</span> <span class="fn">generate_content_async</span>(self, llm_request, **kwargs) -> LlmResponse:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)